### DISCUSSION TITLE
添加多个api接口并设置优先级

### DISCUSSION THREAD

**Discussion by JavaZeroo** [immersive-translate/immersive-translate#1426] [Ideas]
### 新功能 | New Feature

1. 支持添加多个API
2. 手动设置优先级等等。

### 为什么您想要添加这些功能 | Why you want to add these features

1. DeepL API给了很多，大部分情况下都不会用完，但还是会有几个月有用完的情况

**Comment by theowenyoung** [comment]
不同的key用 , 分割即可。

**Reply by JavaZeroo** [reply]
感谢回复，那可以使用多个“翻译服务”吗？因为DeepL API的账号不是很好弄😧

**Reply by theowenyoung** [reply]
应该不会添加，逻辑上有点绕，会让整个插件逻辑变得复杂。
